pkg/inst/tests/test_utils.R

```diff
@@ -35,3 +35,51 @@ test_that("reserialize on RDD", {
   
   unlink(fileName)
 })
+
+test_that("cleanClosure on R functions", {
```

Comment:
Could we add a separate test case that mimics the bug in SPARKR-196 ? (i.e. using `t` as a variable and running lapply)